recipes/hairsplitter/meta.yaml

```diff
@@ -0,0 +1,45 @@
+{% set name = "HairSplitter" %}
+{% set version = "1.7.13" %}
+{% set sha256 = "4387ae2076fd4b8b5e5610ad3a8e678979522403a4735fa4a0522db2feb653f7" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/RolandFaure/HairSplitter/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number : 0
+
+requirements:
+  build:
+    - cmake
+    - make
+    - openmp
+    - {{ compiler('cxx') }}
+  run_exports:
+    - {{ pin_subpackage("hairsplitter", max_pin="2") }}
+  run:
+    - python
+    - scipy
+    - numpy
+    - minimap2
+    - minigraph >=0.20
+    - racon
+    - samtools >=1.16
+    - openmp
+    - raven-assembler
+
+test:
+  commands:
+    - hairsplitter.py --help
+
+about:
+  home: https://github.com/RolandFaure/HairSplitter
+  license: GPL3
+  license_file: LICENSE
+  summary: "Recovers collapsed haplotypes from a draft assembly and long reads"
+  maintainer : RolandFaure # Optional
```

Comment:
Maybe add this add the recipe maintainer under the `extra` section ?
e.g. https://github.com/bioconda/bioconda-recipes/blob/768c14d99cc274506da5b049c06780893017c58f/recipes/snakemake/meta.yaml#L65